### ISSUE TITLE
Hiding some content

### ISSUE THREAD

**Issue by TorgovetsSKotom** [pulsar-edit/pulsar#797]
### Thanks in advance for your bug report!

- [X] Have you reproduced issue in [safe mode](https://pulsar-edit.dev/docs/atom-archive/faq/sections/what-does-safe-mode-do.html)?
- [X] Have you used the [debugging guide](https://pulsar-edit.dev/docs/atom-archive/hacking-atom/#debugging) to try to resolve the issue?
- [X] Have you checked our [FAQs](https://pulsar-edit.dev/docs/launch-manual/sections/faq/) to make sure your question isn't answered there?
- [X] Have you checked to make sure your issue does not already [exist](https://github.com/issues?q=is%3Aissue+user%3Apulsar-edit+is%3Aopen)?
- [X] Have you checked you are on the latest release of Pulsar?

### What happened?

For such sections of the window as Linter/KeyBinding Resolver and others, part of the text is hidden and scrolling is possible in a certain situation

### Pulsar version

1.110.0

### Which OS does this happen on?

🪟 Windows

### OS details

Windows 10

### Which CPU architecture are you running this on?

x86_64/AMD64

### What steps are needed to reproduce this?

1. In windows like Linter/Key Binding Resolver, get enough content that requires vertical scrolling. (if necessary, reducing the height of the main program window)
2. Try to stretch the output window to the entire height of the window.
3. If you drag above the maximum height, the output list will scroll a little and the bottom lines will not be visible. The scroll bar will disappear.
4. This can be fixed by returning the window height

**Comment by confused-Techie** [comment]
Thanks a ton for reporting this issue!

I'll be honest I had to push things to the extreme to recreate this issue within the Key Binding resolver window, but I'm not sure which linter you are using? Does this linter also create a pane in the bottom dock? Since this could just mean it's an issue with how Pulsar as a whole renders the bottom dock.

Where it seems from what I was able to gather, that the very bottom status bar is considered part of the height of the dock at a small scale, meaning part of the scroll window renders underneath it

**Comment by TorgovetsSKotom** [comment]
Yes, indeed, in most use cases, this problem does not occur and it is possible to work.
Yes, linter creates a bottom dock. Standard [linter](https://web.pulsar-edit.dev/packages/linker) in conjunction with [pulsar-ide-python](https://web.pulsar-edit.dev/packages/pulsar-ide-python).
I don't think the problem is in the status bar. Everything that does not fit is hidden. The feeling that the bottom dock does not have a limited upper limit of height. When it reaches its maximum, it still tries to expand and the lower limit goes far down.

**Comment by TorgovetsSKotom** [comment]
"atom-dock-mask" and "atom-dock-content-wrapper bottom" increase their height without restrictions.
